.github/workflows/go.yml

```diff
@@ -43,12 +43,9 @@ jobs:
     - name: Setup Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.17.x
+        go-version: 1.18.x
 
-    - name: Generate
-      run: make --always-make build
-
-    - name: Git Status
+    - name: Ensure stubs are up to date
```

Comment:
This name isn't correct: more than just stub files are generated.

Reply:
Done